isPrimeNumber.js

```diff
@@ -0,0 +1,8 @@
+function isPrimeNumber(number) {
+  for (i = 2; i <= number - 1; i++) {
+    if (number % i == 0) {
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
Good, but it can be improved by looping up to the square root of the input number